### ISSUE TITLE
Custom guild cosmetics

### ISSUE THREAD

**Issue by ForTheWout** [ForTheCrown/FTC#5]
- [ ] ~~Unlockable feature in `Format options`-menu that can only be unlocked via webstore.~~
- [ ] ~~Should be able to be anything like `Xx-&eQues&atmoo&lr-xX` (independent of guild colors)~~
- [ ] ~~Clicking inv-option prompts user to type the guildname~~
- [ ] ~~Errors when name does not contain the guild's name~~

**Comment by JulieWoolie** [comment]
I disagree with this one a bit, I think it would ruin the point of having name formats and unlockable colors if, instead of investing into those, you could just invest in a custom format that let you use whatever formatting/colors you wanted.  

People would just go for it, instead of the unlockable colors and formatting options

**Comment by ForTheWout** [comment]
Maybe instead of choosing the entire format through one pay-walled option, we can just have several extra bracket-formats in the webstore?
